### DISCUSSION TITLE
Remind users to check firewall on first start

### DISCUSSION THREAD

**Discussion by Compositr** [AdguardTeam/AdGuardHome#4086] [Ideas]
I think the [first start](https://github.com/AdguardTeam/AdGuardHome/wiki/Getting-Started#first-time) guide in the wiki could be improved by reminding users to check their firewall, if they cannot access the page, and also remind users to allow traffic from port 53.

I don't know if this is a common issue or if it's just me, but when I was installing Adguard Home on my Raspberry Pi, I wasted a lot of time debugging and checking syslog, just to find out that ufw was blocking requests to port 53, making the DNS unresponsive

**Comment by ainar-g** [comment]
Hello and thanks for the suggestion!  We have now added a paragraph about firewalls to the [First Start section][1].

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/Getting-Started#first-time
